filament/src/View.cpp

```diff
@@ -555,7 +555,7 @@ void FView::computeVisibilityMasks(
         uint8_t visibleLayers,
         uint8_t const* UTILS_RESTRICT layers,
         FRenderableManager::Visibility const* UTILS_RESTRICT visibility,
-        uint8_t* UTILS_RESTRICT visibleMask, size_t count) {
+        uint8_t* UTILS_RESTRICT visibleMask, size_t count, bool hasVsm) {
```

Comment:
please make sure that adding this `bool` doesn't impact how this loop is vectorized.

Reply:
Sounds good, will do.

Reply:
It wasn't getting vectorized, but splitting the statement up into individual booleans did the trick.